src/backend/LamaAPI/Application/Photos/Events/PhotoDeletedEventHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Events.Photo;
+
+using EventBus.Abstraction.Interfaces;
+
+using System.Threading.Tasks;
+
+using Application.Common.Interfaces;
+
+namespace Application.Photos.Events
+{
+    public class PhotoDeletedEventHandler : IIntegrationEventHandler<PhotoDeletedEvent>
+    {
+        private readonly IApplicationDbContext _context;
+
+        public PhotoDeletedEventHandler(IApplicationDbContext context)
+        {
+            _context = context;
+        }
+
+        public Task Handle(PhotoDeletedEvent integrationEvent)
+        {
+            System.Console.WriteLine($"PHOTO DELETED EVENT {integrationEvent.PhotoId}");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/iamprovidence/Lama/pullRequest?prid=4794247)